main.nf

```diff
@@ -38,26 +38,27 @@ def helpMessage() {
                                       Works also with the path to a directory on mapping step with a single germline sample only
                                       Alternatively, path to VCF input file on annotate step
                                       Multiple VCF files can be specified with quotes
-      --step                   [list] Specify starting step
+      --step                   [list] Specify starting step (only one)
                                       Available: mapping, prepare_recalibration, recalibrate, variant_calling, annotate, Control-FREEC
-                                      Default: mapping
+                                      Default: ${params.step}
```

Comment:
I'm not sure that this is a good idea.. If I run `nextflow run --help --step fubar` then it will say that the default is `fubar`? Is there a reason not to hardcode it?

Note that with the JSON schema this will be refactored anyway I guess..

Reply:
Also seems odd to do this for some options and not others?

Reply:
Figured it all get better with the JSON schema refactorization anyway